### ISSUE TITLE
Feature Request: Outfits that kill/disable crewmembers

### ISSUE THREAD

**Issue by taskinprogress** [endless-sky/endless-sky#4099]
Would it be possible/worthwhile to create an attribute that allows non-h2h weapons/outfits to kill (or disable) crewmembers? I think weapons that do so would be pretty self-explanatory, but probably hard to balance. But perhaps a more interesting use would be to add outfits that could benefit crews of certain species while severely punishing those who try to capture ships that carry them.

For example: the Quarg ships might include a massless outfit that automatically lowers the air pressure in their ships to better simulate their natural living conditions. This outfit could allow additional cooling, increased energy, shield/hull repair, etc. or whatever makes sense for the specific outfit. However, if a player attempts to capture the ship, the decreased air pressure would make it difficult for crew members (human presumably) to survive, so a random number could die/pass out every n seconds. If this outfit were tagged as unplunderable, the ship would need to be dragged back to an outfitter to remove the outfit. Ultimately, this would make it far harder to capture these ships, even if they can be disabled and captured, as the captured ships become harder to control and prime targets for being blown up by whatever faction you capture them from.

Another use might be to slightly balance the h2h defense systems. For example, with the Pug biodefenses, the Pug are completely immune, but if a player uses them, there is a small chance that some number of crew on the player's ship will also die. Likewise for the Quarg intrusion countermeasures, which could make the security station less unfavorable.

As I've described it, this attribute (or possibly attributes) would need to recognize an (optional) immune species or faction and some kind of death/disabling rate. As far as I can tell, this would require at least a few changes to the source code.

**Comment by tehhowch** [comment]
Before anything like this could be added, first a more comprehensive crew system would be needed. And this is not really meant to be a crew micromanagement game such as FTL.

**Comment by 10010101001** [comment]
Quarg outfit is called Intrusion Countermeasures. It depressurizes the cabin where enemy crews are boarding.

**Comment by Brick63** [comment]
Nope, it's a random number of crew between Max and required with government not mattering very much

**Comment by Amazinite** [comment]
I'm in agreement with tehhowch. I don't think that crew should be used for a resource for outfits in this way like energy or fuel is.
